app/admin/dashboard.rb

```diff
@@ -440,15 +440,15 @@ def export_metrics
         end # panel
 
         panel 'Practice Engagement' do
```

Comment:
change to innovation here. I'm noticing the `admin` path in general does not have "practice" changed to "innovation"

Reply:
Got it... updated other labels accordingly.